src/models/pokestop.js

```diff
@@ -30,6 +30,7 @@ class Pokestop extends Model {
         'incidentExpireTimestamp',
         'pokestopDisplay',
         'gruntType',
+        'isArScanEligible',
```

Comment:
```suggestion
        'arScanEligible',
```

Reply:
Personally I think `isArScanEligible` is a better name. `is` indicates that this is a Boolean variable.